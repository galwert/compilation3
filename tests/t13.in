int func1(int param1)
{
    return 1;
}

int main() // this is not the right main!!
{
    return 2;
}

// Should fail, no VOID main function