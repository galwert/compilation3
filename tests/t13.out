---end scope---
param1 INT -1
---end scope---
Program has no 'void main()' function
